Ginger/Ginger/Actions/ActionEditPages/ActFileTransferEditPage.xaml.cs

```diff
@@ -71,15 +71,15 @@ private void BrowsePCPathButton_Click(object sender, RoutedEventArgs e)
             System.Windows.Forms.OpenFileDialog dlg = new System.Windows.Forms.OpenFileDialog();
             dlg.DefaultExt = "*.*";
             dlg.Filter = "Any Data Files (*.*)|*.*";
-            string SolutionFolder =  WorkSpace.Instance.Solution.Folder.ToUpper();
+            string SolutionFolder = WorkSpace.Instance.Solution.Folder;
 
             if (dlg.ShowDialog() == System.Windows.Forms.DialogResult.OK)
             {
                 // replace Absolute file name with relative to solution
-                string FileName = dlg.FileName.ToUpper();
-                if (FileName.Contains(SolutionFolder))
+                string FileName = dlg.FileName;
+                if (FileName.ToUpper().Contains(SolutionFolder.ToUpper()))
                 {
-                    FileName = FileName.Replace(SolutionFolder, @"~\");
+                    FileName = @"~\" + FileName.Remove(0, SolutionFolder.Length);
```

Comment:
why you preferred Remove than replace?

Let's use centalized function (need to check if exist and if not so to add one) which deals with relative path handling instead doing it inside the Action code

Reply:
1. not to be depend on case sensitive
2. ok, we have one